<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PCAppsExperiment</class>
 <widget class="QMainWindow" name="PCAppsExperiment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>841</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PCAppsExperiment</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>731</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_color">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>323</width>
      <height>243</height>
     </rect>
    </property>
    <property name="sceneRect">
     <rectf>
      <x>0.000000000000000</x>
      <y>0.000000000000000</y>
      <width>320.000000000000000</width>
      <height>240.000000000000000</height>
     </rectf>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_depth">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>530</y>
      <width>323</width>
      <height>243</height>
     </rect>
    </property>
    <property name="sceneRect">
     <rectf>
      <x>0.000000000000000</x>
      <y>0.000000000000000</y>
      <width>320.000000000000000</width>
      <height>240.000000000000000</height>
     </rectf>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_resetView">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>530</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>ResetView</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_virtual_depth">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>490</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Virtual Depth</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>570</y>
      <width>121</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox { 
     border: 1px solid gray; 
     border-radius: 3px; 
 }
QGroupBox::title { 
    background-color: transparent;
     subcontrol-position: top left; /* position at the top left*/ 
     padding:2 5px;
 } </string>
    </property>
    <property name="title">
     <string>ViewOptions</string>
    </property>
    <widget class="QCheckBox" name="checkBox_normal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Normal</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_view_color">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_view_our_curvature">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Curvature</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_view_fpfh">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>FPFH</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_view_shot">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>SHOT</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_replay">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>0</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Replay</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_timer">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>30</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Timer</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_dataset">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>60</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_every_dataset">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>100</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>EveryDSet</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>170</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox { 
     border: 1px solid gray; 
     border-radius: 3px; 
 }
QGroupBox::title { 
    background-color: transparent;
     subcontrol-position: top left; /* position at the top left*/ 
     padding:2 5px;
 } </string>
    </property>
    <property name="title">
     <string>Data</string>
    </property>
    <widget class="QRadioButton" name="radioButton_data_scenes">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Scenes</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_data_objects">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>42</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Objects</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkBox_save_descriptors">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>130</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>SaveDescr</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_add_noise">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>260</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Add noise</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>876</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
